locals.tf

```diff
@@ -40,10 +36,17 @@ nginx:
         key: sandbox.gke.io/runtime
         operator: Equal
         value: gvisor
-
 astronomer:
   houston:
     config:
+<<<<<<< HEAD
+    %{if var.smtp_uri != ""}
+        email:
+          enabled: true
+          smtpUrl: ${var.smtp_uri}
+    %{endif}
+=======
+>>>>>>> master
```

Comment:
nope

Reply:
I used my atom's GUI git thing to get rid of that - not sure how it got back in - just resolved.